ppx/html.ml

```diff
@@ -15,558 +17,1859 @@ type eventType =
+  ; Attribute
+      {name= "xHeight"; htmlName= "xHeight"; type_= String (* number |  *)}
+  ; Attribute {name= "xlinkActuate"; htmlName= "xlinkActuate"; type_= String}
+  ; Attribute {name= "xlinkArcrole"; htmlName= "xlinkArcrole"; type_= String}
+  ; Attribute {name= "xlinkHref"; htmlName= "xlinkHref"; type_= String}
+  ; Attribute {name= "xlinkRole"; htmlName= "xlinkRole"; type_= String}
+  ; Attribute {name= "xlinkShow"; htmlName= "xlinkShow"; type_= String}
+  ; Attribute {name= "xlinkTitle"; htmlName= "xlinkTitle"; type_= String}
+  ; Attribute {name= "xlinkType"; htmlName= "xlinkType"; type_= String}
+  ; Attribute {name= "xmlBase"; htmlName= "xmlBase"; type_= String}
+  ; Attribute {name= "xmlLang"; htmlName= "xmlLang"; type_= String}
+  ; Attribute {name= "xmlns"; htmlName= "xmlns"; type_= String}
+  ; Attribute {name= "xmlnsXlink"; htmlName= "xmlnsXlink"; type_= String}
+  ; Attribute {name= "xmlSpace"; htmlName= "xmlSpace"; type_= String}
+  ; Attribute {name= "y1"; htmlName= "y1"; type_= String (* number |  *)}
+  ; Attribute {name= "y2"; htmlName= "y2"; type_= String (* number |  *)}
+  ; Attribute {name= "y"; htmlName= "y"; type_= String (* number |  *)}
+  ; Attribute
+      {name= "yChannelSelector"; htmlName= "yChannelSelector"; type_= String}
+  ; Attribute {name= "z"; htmlName= "z"; type_= String (* number |  *)}
+  ; Attribute {name= "zoomAndPan"; htmlName= "zoomAndPan"; type_= String} ]
+
+let webViewHTMLAttributes =
+  [ Attribute {name= "allowFullScreen"; htmlName= "allowfullscreen"; type_= Bool}
+  ; Attribute {name= "allowpopups"; htmlName= "allowpopups"; type_= Bool}
+  ; Attribute {name= "autoFocus"; htmlName= "autofocus"; type_= Bool}
+  ; Attribute {name= "autosize"; htmlName= "autosize"; type_= Bool}
+  ; Attribute {name= "blinkfeatures"; htmlName= "blinkfeatures"; type_= String}
+  ; Attribute
+      { name= "disableblinkfeatures"
+      ; htmlName= "disableblinkfeatures"
+      ; type_= String }
+  ; Attribute
+      {name= "disableguestresize"; htmlName= "disableguestresize"; type_= Bool}
+  ; Attribute
+      {name= "disablewebsecurity"; htmlName= "disablewebsecurity"; type_= Bool}
+  ; Attribute {name= "guestinstance"; htmlName= "guestinstance"; type_= String}
+  ; Attribute {name= "httpreferrer"; htmlName= "httpreferrer"; type_= String}
+  ; Attribute {name= "nodeintegration"; htmlName= "nodeintegration"; type_= Bool}
+  ; Attribute {name= "partition"; htmlName= "partition"; type_= String}
+  ; Attribute {name= "plugins"; htmlName= "plugins"; type_= Bool}
+  ; Attribute {name= "preload"; htmlName= "preload"; type_= String}
+  ; Attribute {name= "src"; htmlName= "src"; type_= String}
+  ; Attribute {name= "useragent"; htmlName= "useragent"; type_= String}
+  ; Attribute {name= "webpreferences"; htmlName= "webpreferences"; type_= String}
+  ]
+
+(* Browser Interfaces https://github.com/nikeee/2048-typescript/blob/master/2048/js/touch.d.ts *)
+(* let abstractView = [] *)
```

Comment:
Why are these commented?

Reply:
Those are artefacts from the copy, most of them are related with types on the event handlers for React. We already type those in events.mli